configs/sklearn_hist_boosting.json

```diff
@@ -0,0 +1,305 @@
+{
+    "common": {
+        "lib": "sklearn",
+        "data-format": "pandas",
+        "data-order": "F",
+        "dtype": "float32",
+        "algorithm": "hist_gb"
```

Comment:
What does `hist_gb` mean? The name should be telling
What about `sklearn_gb` or `skl_gb`?

Reply:
This benchmark test only HistGradientBoostingEstimator from sklearn, so is  sklearn_gb_hist ok?